lib/modules/showImages.js

```diff
@@ -1670,24 +1667,6 @@ addModule('showImages', {
 
 		return Math.round(dragSize);
 	},
-	handleSRStyleToggleVisibility: function(image) {
-		RESUtils.debounce('handleSRStyleToggleVisibility', 50, function() {
-			var toggleEle = modules['styleTweaks'].styleToggleContainer;
-			if (!toggleEle) return;
-			var imageElems = image ? [image] : document.querySelectorAll('.RESImage');
-
-			for (var i = 0; i < imageElems.length; i++) {
-				var imageEle = imageElems[i];
-				var imageID = imageEle.getAttribute('id');
-
-				if (RESUtils.doElementsCollide(toggleEle, imageEle, 15)) {
```

Comment:
`RESUtils.doElementsCollide` is a candidate for removal (I've never seen it anywhere else)